gutenberg.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Gutenberg
  * Plugin URI: https://github.com/WordPress/gutenberg
  * Description: Printing since 1440. This is the development plugin for the new block editor in core. <strong>Meant for development, do not run on real sites.</strong>
- * Version: 1.2.0
```

Comment:
The version bumps here shouldn't be part of the changes.

Reply:
Rebased and removed.